uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/passengerdetails/PassengerDetailsView.kt

```diff
@@ -119,17 +95,11 @@ class PassengerDetailsView @JvmOverloads constructor(
             validationCallback?.onFieldsValidated(areFieldsValid())
         }
 
-        countryCodeSpinner.setOnFocusChangeListener { _, hasFocus ->
-            if (!hasFocus) {
-                if (!arePhoneFieldsValid(PhoneNumberValidator())) {
-                    mobileNumberLayout.error =
-                        resources.getString(R.string.kh_uisdk_invalid_phone_number)
-                } else {
-                    mobileNumberLayout.error = null
-                }
-            }
-
-            validationCallback?.onFieldsValidated(areFieldsValid())
+        countryFlagLayout.setOnClickListener { _ ->
+            val intent = Intent(context, CountryPickerActivity::class.java)
```

Comment:
i think we should be consistent on how to start activities and add a builder pattern to the CountryPickerActivity. An example activity can be RideDetailActivity. You can add the country code in the builder